torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -211,6 +215,17 @@ std::string shapesToStr(const std::vector<std::vector<int64_t>>& shapes) {
   return oss.str();
 }
 
+std::string stacksToStr(const std::vector<std::string>& stacks) {
+  std::ostringstream oss;
+  for (std::vector<std::string>::size_type idx = 0; idx < stacks.size(); ++idx) {
+    if (idx > 0) {
+      oss << ";";
+    }
+    oss << stacks[idx];
+  }
+  return oss.str();
```

Comment:
Can we do it an a bit more C++ way:
```suggestion
  std::copy(stacks.begin(), stacks.end(), std::ostream_iterator<std::string>(oss, ";"));
  auto rc = oss.str();
  rc.pop_back();
  return rc;
```

Reply:
@gaoteng-git could you please update?